typescript/infra/package.json

```diff
@@ -7,6 +7,8 @@
     "@aws-sdk/client-iam": "^3.74.0",
     "@aws-sdk/client-kms": "3.48.0",
     "@aws-sdk/client-s3": "^3.74.0",
+    "@eth-optimism/contracts": "^0.5.39",
+    "@eth-optimism/contracts-bedrock": "^0.13.1",
```

Comment:
why do we need these in infra?

Reply:
I used it for the Predeploys in the tests, buy shouldn't we be using the bedrocks contracts anyways as it will be upgraded to soon?